crates/forge/src/cmd/inspect.rs

```diff
@@ -58,6 +61,9 @@ impl InspectArgs {
             build.compiler.optimize
         };
 
+        // Get the solc version if specified
+        let solc_version = build.use_solc.as_ref().cloned();
```

Comment:
isn't this just clone?
```suggestion
        let solc_version = build.use_solc.clone();
```